docs/zh_cn/getting_started.md

```diff
@@ -27,16 +38,14 @@ title: 安装
    kubeletDir: <kubelet-dir>
    ```
 
-2. 部署
-
-   执行以下命令部署 JuiceFS CSI 驱动：
+1. 安装 CSI 驱动：
 
    ```shell
-   helm repo add juicefs https://juicedata.github.io/charts/
-   helm repo update
    helm install juicefs-csi-driver juicefs/juicefs-csi-driver -n kube-system -f ./values.yaml
    ```
 
+1. 我们推荐将 CSI 驱动的 Helm chart 纳入 Git 仓库，进行源码管理。这样一来，就算 `values.yaml` 中的配置不断变化，也能对其进行追溯和回滚。
```

Comment:
Maybe we can add link of default `values.yaml`(https://github.com/juicedata/charts/blob/main/charts/juicefs-csi-driver/values.yaml) and let users know what this file looks like.